apps/web/src/hooks/useCreateWorkflowFromBlueprint.ts

```diff
@@ -0,0 +1,35 @@
+import slugify from 'slugify';
+import { useMutation, UseMutationOptions } from '@tanstack/react-query';
+import type { INotificationTemplate } from '@novu/shared';
+
+import { useCreateTemplateFromBlueprint } from '../api/hooks';
+import { getBlueprintTemplateById, getTemplateById } from '../api/notification-templates';
+import { TemplateCreationSourceEnum } from '../pages/templates/shared';
+import { getWorkflowBlueprintDetails } from '../utils';
+
+export const useCreateWorkflowFromBlueprint = (
+  options: UseMutationOptions<INotificationTemplate, any, { blueprintIdentifier: string }> = {}
+) => {
+  const { mutateAsync: createTemplateFromBlueprint } = useCreateTemplateFromBlueprint();
+
+  const { mutate: createWorkflowFromBlueprint, ...rest } = useMutation(async ({ blueprintIdentifier }) => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** Prefer more explanatory / semantic name than `rest` por favor‏